_docs/content/en/tutorial/layers.md

```diff
@@ -81,17 +74,31 @@ Notes about the implementation above
 - We provide new GeoJsonLayer with a url to the data.
 - Because this data is in Vancouver, we set the viewState of DeckGL and Mapbox to default to Vancouver so we can see it.
 
-#### Future Implementation of Layers
+#### Are we not currently providing a layer you want to use? Create it and submit a PR! It's easy!
 
-Because we love Declarative Templating so much, our eventual goal is to build small Component wrappers around Layers so that the API is providing Layers via Slot rather than props. These also will include "nice to haves" that make Layers easier to use.
+To do this, all you need to do is create a layer that utilizes the BaseLayerMixin. This mixin is how Vue DeckGL knows that a slotted component passed was a layer type. Here's a quick example of implementing a GeoJsonLayer
 
-An example being...
 ```
 <template>
```

Comment:
I want to try something like this, a "renderless" component here. https://dev.to/ycmjason/vue-without-view-an-introduction-to-renderless-components-23ld

So, we'd end up with something like: ```<script>
import {GeoJsonLayer} from '@deck.gl/layers';
import BaseLayerMixin from './BaseLayerMixin'
export default {
    name: 'GeoJsonLayer',
    mixins:[BaseLayerMixin],
    created(){
        this.layer = new GeoJsonLayer({
            ...this.$attrs,
            data: this.layerData,
        }),
         render: () => null,
    },
}
</script>``` if they're aren't any gotchas with that implementation here.

Reply:
`npm run serve` yields this message for me.  
Command vue serve requires a global addon to be installed.
Please run undefined @vue/cli-service-global and try again.

I am able to run my other new vue project. Fun times. Thought this would be a quick thing to test.

Reply:
My new project, that I can run, uses `    "serve": "vue-cli-service serve",` in `package.json` vue_deckgl uses, `    "serve": "vue serve",` <- I probably did that before I'd upgraded npm, vue cli and all that.